CMakeLists.txt

```diff
@@ -142,6 +142,19 @@ if(${BUILD_BENCHMARKS})
     # surpresses the "'CSVReporter' is deprecated" errors when compiling benchmark.cc
     set_target_properties(${binaryname} PROPERTIES COMPILE_OPTIONS "-Wno-deprecated-declarations" )
   endforeach(binaryfile ${binaries})
+  # For compilation of benchmark_tutorial begin
+  set(binaryfile "${CMAKE_SOURCE_DIR}/examples/benchmark_tutorial.cc")
```

Comment:
You can use `list(APPEND binaries ${CMAKE_SOURCE_DIR}/examples/benchmark_tutorial.cc)` above instead of duplicating the code here =)

Reply:
has been adapted in the `performance/benchmarks/CMakeLists.txt`